spec/models/user_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'Lilly two') }
+
+  it 'name should be present' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'posts counter should be an integer' do
+    subject.posts_counter = 'some_string'
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ ] @Xghayor Great work adding tests for the user model validations, however, you forgot to add tests for the custom method `three_recent_posts` which is part of the requirements
![image](https://github.com/Xghayor/social-media/assets/61408860/1078a536-70eb-4c88-8c7a-3c9e58e800c8)

Reply:
- [ ] @Xghayor Kindly consider adding tests for the comment and like models, this is part of the project requirement as shown below
![image](https://github.com/Xghayor/social-media/assets/61408860/50f2a955-6fa5-4841-8bab-d61c841e3d32)